prepare-image.sh

```diff
@@ -2,7 +2,7 @@
 
 set -euxo pipefail
 
-dnf install -y python3 python3-requests
+dnf install -y python3 python3-requests epel-release
```

Comment:
Please make enabling epel conditional on RESTART_CONTAINER_CERTIFICATE_UPDATED, not everyone wants to have it (and it may even conflict with tripleo repos).

Reply:
@dtantsur The environment variable RESTART_CONTAINER_CERTIFICATE_UPDATED can only be set when we run the container, but the package needs to be installed when the image is built. Do you have any ideas how should we handle it?

Reply:
You can provide options in Dockerfiles as well

Reply:
When we use the image, for example in BMO, we only pull the images from quay.io. We don't rebuild the image using the Dockerfile. In this case, we cannot pick the option that is provided in Dockerfiles.

Reply:
Well, you have to rebuild it or publish an alternative image. Or you can only install inotify from EPEL without pulling in anything else.
```
09:45 <dtantsur> hi folks! I remember long ago it was not recommended to use EPEL with OpenStack repositories. What's the situation now?
10:26 <apevec> dtantsur, the same
```